src/provider_misc.jl

```diff
@@ -26,6 +26,13 @@ function process(r::JSONRPC.Request{Val{Symbol("initialize")},Dict{String,Any}},
     end
     response = JSONRPC.Response(get(r.id), InitializeResult(serverCapabilities))
     send(response, server)
+
+    if isempty(server.cache)
+        send(Message(3, "Building cache, this may take a few minutes"), server)
+        run(`julia -e "using LanguageServer; top = Dict();LanguageServer.modnames(Main, top); LanguageServer.savecache(top)"`)
```

Comment:
Same here, this should be ``run(`$JULIA_HOME/julia`` etc. to make sure we are calling the same julia binary here.